api/src/test/scala/hmda/api/http/ValidationErrorConverterSpec.scala

```diff
@@ -61,6 +61,13 @@ class ValidationErrorConverterSpec extends WordSpec with MustMatchers with Valid
       infos.last mustBe EditInfo("S100", s100Desc)
     }
 
+    "get msa info for Q595 or Q029" in {
+      val errorQ595 = QualityValidationError("8299422144", "Q595", ts = false)
+      val result = validationErrorToResultRow(errorQ595, validationState)
+      val msaField = result.fields.getFields("Metropolitan Statistical Area / Metropolitan Division Name").head.toString
+      msaField mustBe "\"Battle Creek, MI\""
```

Comment:
I believe it's just because of the conversion, the actual API responses don't have any extra quotation marks.